### PR TITLE
fix #2703

### LINKED ISSUE #2703: google maps error on stats page
Google maps window on `https://qiita.ucsd.edu/stats/` shows an error: `This page can't load Google Maps correctly.`

### PR DESCRIPTION
**Pull request by antgonza**
This is an example of how it looks with 2 pointers:
new-map

Note the new qiita-pointers! 
qiita-pointer

### CODE REVIEW

File: qiita_pet/templates/stats.html [hunk 1]
```diff
@@ -1,50 +1,62 @@
...
+    var iconStyle = new ol.style.Style({
+      image: new ol.style.Icon(/** @type {olx.style.IconOptions} */ ({
+        anchor: [0.5, 46],
+        anchorXUnits: 'fraction',
+        anchorYUnits: 'pixels',
+        opacity: 0.75,
+        src: '{% raw qiita_config.portal_dir %}/static/img/qiita-pointer.png'
+      }))
+    });
+    // internal funtion to avoid duplication of code
```
Comment (ElDeveloper) [inline qiita_pet/templates/stats.html]: funtion -> function

File: qiita_pet/templates/stats.html [hunk 2]
```diff
@@ -1,50 +1,62 @@
 {% extends sitebase.html %}
 {% block head %}
+<link rel="stylesheet" href="https://cdn.rawgit.com/openlayers/openlayers.github.io/master/en/v5.2.0/css/ol.css" type="text/css">
```
Comment (ElDeveloper) [inline qiita_pet/templates/stats.html]: Can you also add this to the project?

**Comment by wasade** [comment]
Looks good!

**Comment by antgonza** [comment]
Looks like we have 2 reviews, can this be merged?

**Pull request merged**